frontend/app/features/kanban/components/KanbanBoard.tsx

```diff
@@ -1,21 +1,87 @@
 "use client";
 
+import { useGetAllTasksQuery } from "@store/api/taskSlice";
+import { useState } from "react";
 import styles from "../styles/kanban.module.css";
+import { filterTasks, TaskFilters } from "../utils/filterTasks";
 import CreateTaskForm from "./CreateTaskForm";
+import TaskFilter from "./TaskFilter";
 
 const KanbanBoard: React.FC = () => {
+  const { data: tasks = [], error, isLoading } = useGetAllTasksQuery();
+  const [filters, setFilters] = useState<TaskFilters>({
+    status: "ALL",
+    createdBy: "ALL",
+    assignedTo: "ALL",
+    dueDate: "ALL",
+    createdAt: "ALL",
+    createdAtOrder: "NEWEST",
```

Comment:
use ASC and DESC, thats the common term we use for it, its also compatible with BE